conformance_pack/lambda.sp

```diff
@@ -365,12 +365,12 @@ query "lambda_function_use_latest_runtime" {
       arn as resource,
       case
         when package_type <> 'Zip' then 'skip'
-        when runtime in ('nodejs18.x', 'nodejs16.x', 'nodejs14.x', 'python3.10', 'python3.9', 'python3.8', 'python3.7', 'ruby3.2', 'ruby2.7', 'java17', 'java11', 'java8', 'java8.al2', 'go1.x', 'dotnet7', 'dotnet6') then 'ok'
+        when runtime in ('nodejs20.x', 'nodejs18.x', 'nodejs16.x', 'python3.12', 'python3.11', 'python3.10', 'python3.9', 'python3.8', 'ruby3.2', 'java21', 'java17', 'java11', 'java8.al2', 'dotnet8', 'dotnet7', 'dotnet6', 'provided.al2023', 'provided.al2') then 'ok'
```

Comment:
@sbldevnet  Thank you for catching this bug and submitting the PR. We just checked the AWS FSBP [document](https://docs.aws.amazon.com/securityhub/latest/userguide/lambda-controls.html#lambda-2), and it does not completely match the changes made. We strictly adhere to this FSBP document. According to this document, below are the list of runtimes. Could you please update the query to align with the document?
Thanks!!
```
runtime: dotnet8, dotnet6, java21, java17, java11, java8.al2, nodejs20.x, nodejs18.x, nodejs16.x, python3.12, python3.11, python3.10, python3.9, python3.8, ruby3.3, ruby3.2
```

Reply:
Updated